server/models/Squashes.js

```diff
@@ -0,0 +1,10 @@
+var Promise = require('bluebird');
+var Squashes = module.exports;
+
+
+var Squashes =  ['drank', 'drunk', 'deployed', 'got', 'developed', 'built', 'invented', 'experienced', 'fought off', 'hardened', 'enjoyed', 'developed', 'consumed', 'debunked', 'drugged', 'doped', 'made', 'wrote', 'saw'];
```

Comment:
To make this a little more realistic (and useable by the frontend), make each quash an object with two properties: `title` and `content`